src/protagonist/API.Tests/Integration/ModifyAssetTests.cs

```diff
@@ -687,19 +685,19 @@ public async Task Put_NewAudioAsset_Creates_Asset()
         // assert
         response.StatusCode.Should().Be(HttpStatusCode.Created);
         response.Headers.Location.PathAndQuery.Should().Be(assetId.ToApiResourcePath());
-        var asset = dbContext.Images.Include(i => i.ImageDeliveryChannels).Single(x => x.Id == assetId);
+        var asset = dbContext.Images.Include(i => i.ImageDeliveryChannels)
+            .ThenInclude(dc => dc.DeliveryChannelPolicy).Single(x => x.Id == assetId);
         asset.Id.Should().Be(assetId);
         asset.MaxUnauthorised.Should().Be(-1);
         asset.ImageDeliveryChannels.Count.Should().Be(1);
         asset.ImageDeliveryChannels.Should().ContainSingle(x => x.Channel == "iiif-av" &&
-                                                                x.DeliveryChannelPolicyId == 5);
+                                                                x.DeliveryChannelPolicy.Name == "default-audio");
```

Comment:
I think I also changed this line in PR that was just merged to `develop` so you may have a few conflicts.